dll/win32/ws2_32/src/startup.c

```diff
@@ -237,6 +342,30 @@ WSAStartup(IN WORD wVersionRequested,
     /* Check if all worked */
     if (ErrorCode == ERROR_SUCCESS)
     {
+	/* create events for functio WSAThread */
```

Comment:
Typo fix:
`functio` -> `function`

Suggestion:
`/* create events for WSAThread function */`

Reply:
You are right :)